tests/midlets/background/PauseMIDlet.java

```diff
@@ -0,0 +1,32 @@
+package tests.background;
```

Comment:
(NB: I wonder if we should distinguish between midlets that are intended to be run manually and those that support automated tests. Right now we mix both in tests/midlets/.)